modules/linear_boltzmann_solvers/discrete_ordinates_solver/sweep/fluds/aah_fluds_common_data.cc

```diff
@@ -181,7 +181,8 @@ AAH_FLUDSCommonData::SlotDynamics(const Cell& cell,
         bool found = false;
         for (auto& lock_box_slot : lock_box)
         {
-          if ((lock_box_slot.first == face.neighbor_id_) and (lock_box_slot.second == ass_face))
+          if ((static_cast<uint64_t>(lock_box_slot.first) == face.neighbor_id_) and
+              (lock_box_slot.second == ass_face))
```

Comment:
@wdhawkins @zachhardy does anyone know what ass_face is, except sounding funny to a 4-year old.

Reply:
Yes, I believe it's the cell neighbor face that is associated with this cell's face (hence ass_face = associated face).

Reply:
I am not so sure because we use a`dj_face, adj_cell`, where adj = adjacent. So, ass_face may be different than adj_face. Exampl using adj_ everywhere:

```
int af = -1;
for (const auto& adj_face : adj_cell.faces_)
{
++af;
std::set<uint64_t> afvids(adj_face.vertex_ids_.begin(),
 adj_face.vertex_ids_.end()); // adj_face vertex ids
```

Reply:
I had this discussion with Jan in the past. @wdhawkins is correct. It is associated face.